### DISCUSSION TITLE
How to retrieve number of rows affected along with # of rows matched?

### DISCUSSION THREAD

**Discussion by jonataspc** [dbeaver/dbeaver#35053] [General]
Hello,
New Dbeaver user here.
Is there a way to configure DBeaver to show both # of rows affected and also # of rows matched after running an UPDATE statement?

This is available in MySQL Workbench:

![image](https://github.com/user-attachments/assets/2a564f53-538a-4cee-979b-b22af49cf20b)

**Comment by E1izabeth** [comment]
Currently, DBeaver shows only affected rows. However, you can create a feature request :)

**Reply by jonataspc** [reply]
Done, https://github.com/dbeaver/dbeaver/issues/35063
